scripts/kconfig/kconfigfunctions.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2018-2019 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import os
+
+# Types we support
+# 'string', 'int', 'hex', 'bool'
+conf_file = os.path.join(os.environ['PROJECT_BINARY_DIR'], "include/generated/generated_dts_board.conf")
+
+dt_defines = {}
+if os.path.isfile(conf_file):
+    for line in open(conf_file, 'r', encoding='utf-8'):
+        if '=' in line:
+            c = line.split('=')
+            dt_defines[c[0]] = c[1].strip()
+
+def _dt_units_to_scale(unit):
+    if not unit:
+        return 0
+    if unit == 'k' or unit == 'K':
+        return 10
+    if unit == 'm' or unit == 'M':
+        return 20
+    if unit == 'g' or unit == 'G':
+        return 30
```

Comment:
Just a comment, but can do `if unit in {'k', 'K'}:`, etc., too.

Reply:
Done